modules/custom/az_publication/config/install/views.view.az_publications.yml

```diff
@@ -663,7 +663,7 @@ display:
           exposed: false
           draggable_views_reference: 'az_publications_curate:curation_page'
           draggable_views_null_order: before
-          draggable_views_pass_arguments: true
+          draggable_views_pass_arguments: 1
```

Comment:
Confirmed that curation contextual links still seem to have argument context after this change (which seems like views config drift)

![image](https://github.com/user-attachments/assets/4541d621-4d42-4671-ac44-b5967624d189)